ecs-microservice/variables.tf

```diff
@@ -261,8 +249,14 @@ variable "grafana_create_dashboard" {
   default     = false
 }
 
+variable "grafana_sns_topic_names" {
```

Comment:
du trenger ikke innføre egne parametrer for å gi inn sns/sqs etc lister det finnes allerede på modulen som
`variable "sns_publish_topics" {
  type    = list(string)
  default = []
}

variable "sns_subscribe_topics" {
  type    = list(string)
  default = []
}`

Reply:
i tillegg finnes relevante parametrer som 
```

variable "dynamodb_tables" {
  type    = list(string)
  default = []
}
```
og
```
variable "sqs_queues" {
  type    = list(string)
  default = []
}
```

Reply:
og enda mer 
```

variable "s3_read_buckets" {
  type    = list(string)
  default = []
}

variable "s3_write_buckets" {
  type    = list(string)
  default = []
}
```

Reply:
jeg tenker det er bedre å bruke dem direkte, istedet for å legge på et duplikat sett med parametrer

Reply:
vi har også 
```
variable "use_elasticache" {
  description = "Use elasticache for caching in microservice"
  default     = 0
}
```

men usikker på hva man kan gjøre, det er samme som rds ikke nevnt en spesifikk cache.
kanskje man kan fiske fram den fra cache resourcen i modulen

Reply:
sns_publish_topics blir laget som en liste av sns arn, ikke terraform ressurser.  Selv om jeg har listen over arn, så går det ikke ann å hente den ut igjen som en data ressurs, da data trenger navnet.
Dimensjonen i grafana trenger også navnet på sns topicen, ikke arn'en

Reply:
Fant en måte å trekke ut navnene basert på arn. Oppretter disse som lokale variable og trekker ut navnene som er siste del av arn'en. Bruker disse igjen i et regexp filter for dashboardet :)